API/hermes/hermes.cpp

```diff
@@ -78,6 +87,14 @@ namespace vm = hermes::vm;
 namespace hbc = hermes::hbc;
 using ::hermes::hermesLog;
 
+namespace hermes::node_api {
+  HERMES_WEAK void
```

Comment:
Using a weak implementation here to provide a default throwing implementation, in case the caller does not have the `hermesNodeApi` target linked in.